File: oa-portal/problems/c5-concert-tickets/starters/main.cpp
#include <bits/stdc++.h>
using namespace std;

vector<int> ticketPrices(vector<int> h, const vector<int>& t) {
    // write your code here

    return vector<int>(t.size(), -1);
}

int main() {
    int n, m;
    if (scanf("%d %d", &n, &m) != 2) return 0;
    vector<int> h(n), t(m);
    for (auto &v : h) scanf("%d", &v);
    for (auto &v : t) scanf("%d", &v);

    string out;
    for (int v : ticketPrices(h, t)) { out += to_string(v); out += '\n'; }
    fwrite(out.data(), 1, out.size(), stdout);
    return 0;
}

File: oa-portal/problems/c5-concert-tickets/starters/main.java
import java.io.*;
import java.util.*;

public class Main {

    static int[] ticketPrices(int[] h, int[] t) {
        // write your code here

        int[] paid = new int[t.length];
        Arrays.fill(paid, -1);
        return paid;
    }

    public static void main(String[] args) throws IOException {
        FastReader in = new FastReader();
        int n = in.nextInt(), m = in.nextInt();
        int[] h = new int[n], t = new int[m];
        for (int i = 0; i < n; i++) h[i] = in.nextInt();
        for (int i = 0; i < m; i++) t[i] = in.nextInt();

        StringBuilder sb = new StringBuilder();
        for (int v : ticketPrices(h, t)) sb.append(v).append('\n');
        System.out.print(sb);
    }

    static class FastReader {
        private final DataInputStream in = new DataInputStream(new BufferedInputStream(System.in, 1 << 16));
        int nextInt() throws IOException { return (int) nextLong(); }
        long nextLong() throws IOException {
            int b = in.read();
            while (b != '-' && (b < '0' || b > '9')) b = in.read();
            boolean neg = b == '-';
            if (neg) b = in.read();
            long v = 0;
            while (b >= '0' && b <= '9') { v = v * 10 + (b - '0'); b = in.read(); }
            return neg ? -v : v;
        }
        String next() throws IOException {
            int b = in.read();
            while (b <= ' ') b = in.read();
            StringBuilder sb = new StringBuilder();
            while (b > ' ') { sb.append((char) b); b = in.read(); }
            return sb.toString();
        }
    }
}
